### ISSUE TITLE
[a11y] connect question heading, description with question input

### ISSUE THREAD

**Issue by susnux** [nextcloud/forms#3172]
**Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue. Please don't comment if you have no relevant information to add!**

**Describe the bug**
Question input fields are not connected with the headings and descriptions.

**Expected behavior**
the input fields should be connected to heading and description (aria-labelledby and aria-describedby).

**Screenshots**

<img width="770" height="347" alt="Image" src="https://github.com/user-attachments/assets/23cfa1c8-5222-40aa-9f2c-fc6fdfc65d68" />
